docs/intro/reusable-apps.txt

```diff
@@ -198,91 +198,60 @@ this. For a small app like polls, this process isn't too difficult.
    license. Just be aware that your licensing choice will affect who is able
    to use your code.
 
-#. Next we'll create ``pyproject.toml``, ``setup.cfg``, and ``setup.py`` files
-   which detail how to build and install the app. A full explanation of these
-   files is beyond the scope of this tutorial, but the `setuptools
-   documentation <https://setuptools.pypa.io/en/latest/>`_ has a good
-   explanation. Create the ``django-polls/pyproject.toml``,
-   ``django-polls/setup.cfg``, and ``django-polls/setup.py`` files with the
-   following contents:
+#. Next we'll create the ``pyproject.toml`` file which details how to build and
+   install the app. A full explanation of this file is beyond the scope of this
+   tutorial, but the `Python Packaging User Guide
+   <https://packaging.python.org/en/latest/>`_ has a good explanation. Create
+   the ``django-polls/pyproject.toml`` file with the following contents:
 
    .. code-block:: toml
-        :caption: ``django-polls/pyproject.toml``
-
-        [build-system]
-        requires = ['setuptools>=40.8.0']
-        build-backend = 'setuptools.build_meta'
-
-   .. code-block:: ini
-        :caption: ``django-polls/setup.cfg``
-
-        [metadata]
-        name = django-polls
-        version = 0.1
-        description = A Django app to conduct web-based polls.
-        long_description = file: README.rst
-        url = https://www.example.com/
-        author = Your Name
-        author_email = yourname@example.com
-        license = BSD-3-Clause  # Example license
-        classifiers =
-            Environment :: Web Environment
-            Framework :: Django
-            Framework :: Django :: X.Y  # Replace "X.Y" as appropriate
-            Intended Audience :: Developers
-            License :: OSI Approved :: BSD License
-            Operating System :: OS Independent
-            Programming Language :: Python
-            Programming Language :: Python :: 3
-            Programming Language :: Python :: 3 :: Only
-            Programming Language :: Python :: 3.10
-            Programming Language :: Python :: 3.11
-            Programming Language :: Python :: 3.12
-            Topic :: Internet :: WWW/HTTP
-            Topic :: Internet :: WWW/HTTP :: Dynamic Content
-
-        [options]
-        include_package_data = true
-        packages = find:
-        python_requires = >=3.10
-        install_requires =
-            Django >= X.Y  # Replace "X.Y" as appropriate
-
-   .. code-block:: python
-        :caption: ``django-polls/setup.py``
-
-        from setuptools import setup
-
-        setup()
-
-#. Only Python modules and packages are included in the package by default. To
-   include additional files, we'll need to create a ``MANIFEST.in`` file. The
-   ``setuptools`` docs referred to in the previous step discuss this file in
-   more detail. To include the templates, the ``README.rst`` and our
-   ``LICENSE`` file, create a file ``django-polls/MANIFEST.in`` with the
```

Comment:
You've removed stuff related to `MANIFEST.in` but that is still applicable when using `setuptools` (although the example here is dated IMO).

Maybe we should recommend using `hatch`/`hatchling` going forward? It keeps all configuration in `pyproject.toml` and also supports most of the functionality of tools like `tox` and `pyenv`.

Reply:
In my testing here, you can drop LICENSE, READAME.rst, and docs from the MANIFEST, but manually including django_polls/static and templates must remain.